LubimovaYE/lab_1/main.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	
+	float operand_1, operand_2;
+	char operation_sign;
+	cout << "Enter 2 numbers and an action sign(+,-,*,/)" << endl;
+	cout << "To exit the program, enter '-1' three times" << endl;
+	cin >> operand_1 >> operand_2 >> operation_sign;
+	while (operand_1 != -1 && operand_2 != -1 && operation_sign != '-1')
```

Comment:
If I try to calculate sum of -1 and 2  your program exits which is not an expected behavior.
![image](https://user-images.githubusercontent.com/19354025/97217742-c62aca80-17d8-11eb-89e6-e9b9d9c9549a.png)